app/_how-tos/configure-oidc-with-password-grant.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Configure OpenID Connect with the password grant
+content_type: how_to
+
+related_resources:
+  - text: Authentication in {{site.base_gateway}}
+    url: /gateway/authentication/
+  - text: OpenID Connect authentication flows and grants
+    url: /plugins/openid-connect/#authentication
+  - text: Password grant workflow
+    url: /plugins/openid-connect/#password-grant-workflow
+
+plugins:
+  - openid-connect
+
+entities:
+  - route
+  - service
+
+products:
+  - gateway
+
+works_on:
+  - on-prem
+  - konnect
+
+min_version:
+  gateway: '3.4'
+
+tools:
+  - deck
+
+prereqs:
+  entities:
+    services:
+      - example-service
+    routes:
+      - example-route
+  inline:
+    - title: Set up Keycloak
+      include_content: prereqs/auth/oidc/keycloak-password
+      icon_url: /assets/icons/keycloak.svg
+
+tags:
+  - authentication
+  - openid-connect
+
+tldr:
+  q: How do I use a username and password to authenticate directly with my identity provider?
+  a: Using the OpenID Connect plugin, set up the [password grant flow](/plugins/openid-connect/#password-grant-workflow) to connect to an identity provider (IdP) by passing a username and password in a header.
+
+cleanup:
+  inline:
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg
+
+---
+
+## 1. Enable the OpenID Connect plugin with the password grant
+
+Using the Keycloak and {{site.base_gateway}} configuration from the [prerequisites](#prerequisites), 
+set up an instance of the OpenID Connect plugin with the password grant.
+
+For the password grant, we need to configure the following:
+* Issuer, client ID, client secret, and client auth: settings that connect the plugin to your IdP (in this case, the sample Keycloak app).
```

Comment:
```suggestion
* `issuer`, `client_id`, `client_secret`, and `client_auth`: settings that connect the plugin to your IdP (in this case, the sample Keycloak app).
```
For some reason it tripped me up that that these were variables without seeing them in the same style as in the deck example.